### ISSUE TITLE
Skip to content

### ISSUE THREAD

**Issue by mattcarlotta** [mattcarlotta/portfolio-astro-vercel#4]
When accessible users are navigating through pages, they should have the ability to skip the top navigation
- Requires adding a skip to content button that sets focus on the `main` HTML element
